tests/unit/gaze/io/csv_test.py

```diff
@@ -51,64 +51,64 @@
+                'pixel_columns': pm.Dataset('HBN', path='').definition.pixel_columns,
             },
             (10, 2),
             id='hbn_dataset_example',
         ),
         pytest.param(
             {
                 'file': 'tests/files/sbsat_example.csv',
-                'time_column': pm.datasets.SBSAT().time_column,
-                'time_unit': pm.datasets.SBSAT().time_unit,
-                'pixel_columns': pm.datasets.SBSAT().pixel_columns,
-                **pm.datasets.SBSAT().custom_read_kwargs['gaze'],
+                'time_column': pm.Dataset('SBSAT', '').definition.time_column,
+                'time_unit': pm.Dataset('SBSAT', '').definition.time_unit,
+                'pixel_columns': pm.Dataset('SBSAT', '').definition.pixel_columns,
+                **pm.Dataset('SBSAT', '').definition.custom_read_kwargs['gaze'],
             },
             (10, 5),
             id='sbsat_dataset_example',
         ),
         pytest.param(
             {
                 'file': 'tests/files/gazebase_example.csv',
-                'time_column': pm.datasets.GazeBase().time_column,
-                'time_unit': pm.datasets.GazeBase().time_unit,
-                'position_columns': pm.datasets.GazeBase().position_columns,
-                **pm.datasets.GazeBase().custom_read_kwargs['gaze'],
+                'time_column': pm.Dataset('GazeBase', '').definition.time_column,
+                'time_unit': pm.Dataset('GazeBase', '').definition.time_unit,
+                'position_columns': pm.Dataset('GazeBase', '').definition.position_columns,
+                **pm.Dataset('GazeBase', '').definition.custom_read_kwargs['gaze'],
             },
             (10, 7),
             id='gazebase_dataset_example',
         ),
         pytest.param(
             {
                 'file': 'tests/files/gaze_on_faces_example.csv',
-                'time_column': pm.datasets.GazeOnFaces().time_column,
-                'time_unit': pm.datasets.GazeOnFaces().time_unit,
-                'pixel_columns': pm.datasets.GazeOnFaces().pixel_columns,
-                **pm.datasets.GazeOnFaces().custom_read_kwargs['gaze'],
+                'time_column': pm.Dataset('GazeOnFaces', '').definition.time_column,
+                'time_unit': pm.Dataset('GazeOnFaces', '').definition.time_unit,
+                'pixel_columns': pm.Dataset('GazeOnFaces', '').definition.pixel_columns,
+                **pm.Dataset('GazeOnFaces', '').definition.custom_read_kwargs['gaze'],
             },
             (10, 1),
             id='gaze_on_faces_dataset_example',
         ),
         pytest.param(
             {
                 'file': 'tests/files/gazebase_vr_example.csv',
-                'time_column': pm.datasets.GazeBaseVR().time_column,
-                'time_unit': pm.datasets.GazeBaseVR().time_unit,
-                'position_columns': pm.datasets.GazeBaseVR().position_columns,
+                'time_column': pm.Dataset('GazeBaseVR', '').definition.time_column,
```

Comment:
you could use `pm.DatasetLibrary.get('GazeBaseVR')` without having to instantiate a `Dataset`

Reply:
yea, will do that instead

Reply:
its a draft and wip, thats why I don't wanna bother yet with stuff and its draft not PR

Reply:
as I said in yesterdays meeting I'll change a lot once everything works the way its supposed to and I want to

Reply:
Don't worry, there's no need to be defensive. It's important to have early feedback in light of our upcoming deadline, otherwise there's the risk of having too much to change in a short time.

Reply:
haha sorry wasn't supposed to be defensive, just wanted to make sure you do not review too much already because I'm in the processing of refactoring everything constantly.

the feedback above is invaluable! <3